src/main/java/com/bootcamp/demo/restapi/ScooterController.java

```diff
@@ -0,0 +1,41 @@
+package com.bootcamp.demo.restapi;
+
+import com.bootcamp.demo.model.Scooter;
+import com.bootcamp.demo.service.ScooterService;
+import com.google.firebase.database.annotations.NotNull;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/api/scooters")
+@Slf4j
```

Comment:
Same here. Use logging differently, not through lombok. There were some resources sent on slack, but if you think further ones are needed, let us know.

Reply:
But it's not clear why not use it?

Reply:
Answered here: https://github.com/nymphadora-tonks-team/booking-jb/pull/20/files#r749681612

Reply:
Fixed. New code available.